template/config/vitest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "test:unit": "vitest --environment jsdom"
+    "test:unit": "vitest --environment jsdom --root src/"
```

Comment:
Why changing this line?

Reply:
Otherwise vitest tries to process the playwright test files as well, this won't work and lead to errors. Maybe there is a better way of handling that?

Reply:
Hello! It seems that this addition introduces a problem. When running Vitest it tries to create a `results.json` inside its cache directory. This directory is defined as `resolve(root, slash(dir || 'node_modules/.vitest'))`. This means that with `--root src/` added, Vitest creates the `results.json` inside the `<project_root>/src/node_modules/.vitest` instead of the actual node_modules directory (`<project_root>/node_modules/.vitest`).